utils/config.env

```diff
@@ -0,0 +1,42 @@
+#####################################################
+# Values in this file will be used to replace variables in script files
+# and Docker Compose files throughout the repo
+#
+# Example for GA release:
+#   CP_VERSION_MAJOR=5.4
+#   CP_VERSION_FULL=5.4.0
+#   GH_BRANCH=5.4.0-post
+#   JAR_VERSION=5.4.0
+#   TAG=5.4.0
+#   REPOSITORY=confluent   # confluentinc repository at https://hub.docker.com/u/confluentinc
+#
+# Example for pre-GA release:
+#   CP_VERSION_MAJOR=5.4
+#   CP_VERSION_FULL=5.4.1
+#   GH_BRANCH=5.4.x
+#   JAR_VERSION=5.4.1-SNAPSHOT
+#   TAG=5.4.x-latest
+#   REPOSITORY=<account>.dkr.ecr.us-west-2.amazonaws.com/confluentinc   # confluentinc repository at the specified ECR registry, images must be pulled separately
+#####################################################
+
+CP_VERSION_MAJOR=5.4
+CP_VERSION_FULL=5.4.0
+GH_BRANCH=5.4.0-post
```

Comment:
_verrrry_ nitpicky, but FYC calling this `GIT_BRANCH` or `CP_BRANCH` as it doesn't really have much to do w/ github in particular... do not hold up the change for this unless you want

Reply:
Ack and done with latest commit.